examples/ios/objc/Draw/Views/SwatchesView.m

```diff
@@ -0,0 +1,138 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2014 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "SwatchesView.h"
+
+static CGFloat kSwatchButtonHeightPhone = 85.0f;
+static CGFloat kSwatchButtonWidthPhone = 30.0f;
+
+static CGFloat kSwatchButtonHeightPad= 166.0f;
+static CGFloat kSwatchButtonWidthPad = 57.0f;
+
+static CGFloat kSwatchPencilPadding = 1.0f;
+
+@interface SwatchesView()
+
+@property (nonatomic, strong) NSDictionary *colors;
+@property (nonatomic, strong) NSArray *colorButtons;
+
+- (void)setupButtons;
```

Comment:
these 4 methods don't need to be declared in this interface extension, they're only used from within this file...